google-cloud-logging/src/main/java/com/google/cloud/logging/spi/v2/GrpcLoggingRpc.java

```diff
@@ -82,7 +81,7 @@ public GrpcLoggingRpc(LoggingOptions options) throws IOException {
       ChannelProvider channelProvider;
       // todo(mziccard): ChannelProvider should support null/absent credentials for testing
       if (options.getHost().contains("localhost")
-          || options.getCredentials().equals(NoCredentials.getInstance())) {
+          || options.credentialIsNoCredentialInstance()) {
```

Comment:
Probably easier:

`NoCredentials.getInstance().equals(options.getCredentials())`

Then you don't need a dedicated method.